.rat-excludes

```diff
@@ -47,3 +47,4 @@ build/scala-*/**
 **/org/apache/kyuubi/ui/static/assets/**
 **/org/apache/kyuubi/ui/swagger/**
 **/org.apache.spark.status.AppHistoryServerPlugin
+**/statestore-schema*.sql
```

Comment:
Why do we exclude SQL files?

Reply:
FYI: https://github.com/apache/dolphinscheduler/blob/2ff76c7e7330c166fba1e514aba6ddf2dbe74f57/dolphinscheduler-dao/src/main/resources/sql/create/release-1.2.0_schema/postgresql/dolphinscheduler_ddl.sql#L1-L16

Reply:
hmm, just remove the licenses to simplify the sql file content(without semicolon).

Reply:
otherwise, we have to truncate the comments or introduce other dependency to split the sql file content for schema initialization.